modules/ROOT/pages/cap-policy.adoc

```diff
@@ -20,7 +20,7 @@ This policy applies to all your APIs behind your Edge/Inbound endpoint. You can
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Value |Description
-|*Maximum Message Size* | Maximum tolerated message size, in bytes, for your policy.
+|*Maximum Message Size* | Maximum tolerated message size, in bytes, for your policy. If request body scanning is enabled, this should be set to a value no larger than what is required to prevent attackers from abusing request body checking and exhausting resources
```

Comment:
"this value should be set no larger than required to prevent...." (since technically "this" isn't clear)